scripts/manipulate_windows_zip.js

```diff
@@ -2,19 +2,21 @@
 
 const spawnSync = require('child_process').spawnSync;
 const path7za = require('7zip-bin').path7za;
-const appVersion = require('../package.json').version;
+const pkg = require('../package.json');
```

Comment:
Please use `require('../src/package.json')`. Actually it affects to artifact name.